### DISCUSSION TITLE
Add "Last Edited" to posts

### DISCUSSION THREAD

**Discussion by trashcatt** [adityatelange/hugo-PaperMod#775] [Ideas]
I'd like to be able to show the date of the last edit to a post on the post (ideally next to the post date).

Is there an easy way to do this already that I'm just overlooking?

**Comment by ichard26** [comment]
Not sure if this counts as your definition of easy but you can override the theme's post_meta.html partial (i.e. the template used to generate the metadata line after the page title / description) to include this information, [here's how I do it on my website](https://github.com/ichard26/ichard26.github.io/blob/652009a0c7f865e74e6ecfe92d7269248e3fff34/layouts/partials/post_meta.html). You can then set the `lastmod` key in the page frontmatter. When both date and lastmod are defined it'll something like this (well the colour change I applied here is separate)
![image](https://user-images.githubusercontent.com/63936253/152722814-af4b3c3a-acbb-4f2b-bb61-eda128d54854.png)

**Reply by d4n3sh** [reply]
I was just thinking of doing this today. Thanks! :D

**Reply by felipecrp** [reply]
👍I created a PR (https://github.com/adityatelange/hugo-PaperMod/pull/1703) based on this suggestion.

**Comment by yyyanbj** [comment]
To use `.Lastmod` parameter, you also need to set `enableGitInfo: true` in your [site’s configuration file](https://gohugo.io/getting-started/configuration/).

**Reply by ichard26** [reply]
You could also set the [lastmod source to use `:fileModTime`](https://gohugo.io/getting-started/configuration/#:~:text=date%20handlers%20are%3A-,%3AfileModTime,-Fetches%20the%20date) instead using Git information like me fwiw.

**Reply by yyyanbj** [reply]
> 

Actually, I use GitHub actions to deploy my blog, so the time of `filemod` is always the deployment time XD. GitInfo is necessary for me.
